ga4gh/gff3.py

```diff
@@ -0,0 +1,223 @@
+"""
+Object representation of GFF3 data and parser for GFF3 files.
+See: http://www.sequenceontology.org/gff3.shtml
+"""
+
```

Comment:
you're missing the division future import here... we want to have all three imports in every python file we create